### PR TITLE
Minor additions of const

### PR DESCRIPTION
**Pull request by tfogal**

### CODE REVIEW

**Comment by tfogal** [comment]
!build

File: csrc/fusion.h [hunk 1]
```diff
@@ -189,8 +189,11 @@ class Fusion : public IrContainer {
   bankConflictInfo(const CompileParams& compile_params = CompileParams());
 
   //! Return a list of topologically sorted expressions. This only includes
-  //! exprs required to genereate registered outputs.
+  //! exprs required to generate registered outputs.
   std::vector<Expr*> exprs();
```
Comment (zasdfgbnm) [inline csrc/fusion.h:193]: Do we still need this non-const version?

**Review by zasdfgbnm — APPROVED** [review]

**Review by jjsjann123 — APPROVED** [review]
LGTM

File: csrc/fusion.h (hunk 1 above)
Comment (tfogal) [reply csrc/fusion.h:193]: hrm, good question! i'll look into that.

Comment (tfogal) [reply csrc/fusion.h:193]: compiles fine locally w/o it. will add you to https://github.com/NVIDIA/Fuser/pull/1798 once it passes CI.

**Pull request merged**